src/hotspot/share/opto/phaseX.cpp

```diff
@@ -1982,7 +1984,7 @@ bool PhaseIterGVN::verify_node_Identity(Node* n) {
     return false;
   }
 
-  // We just saw a new Identity which was not found during IGVN.
+  // The verificatin just found a new Identity that was not found during IGVN.
```

Comment:
```suggestion
  // The verification just found a new Identity that was not found during IGVN.
```
I guess, I suggested a typo...